src/main/java/com/amazon/aocagent/enums/TestCase.java

```diff
@@ -26,6 +26,12 @@
       Arrays.asList(new OTMetricAndTraceEmitterInstaller()),
       Arrays.asList(new MetricValidator(), new TraceValidator())),
 
+  EC2_WIN_TEST(
+      new EC2TestBed(),
+      new OTPackageInstaller(),
```

Comment:
can we implement a "SSMOTPackageInstaller" here and change the test case name to EC2_SSM_TEST, so that in the future we can expand this test case to linux.